### ISSUE TITLE
`cmd:` accepted as task key

### ISSUE THREAD

**Issue by Rots** [go-task/task#911]
As discussed in https://discord.com/channels/974121106208354339/1035111266005569567

With this input
```yml
version: '3'

tasks:
  hello-broken:
    cmd: echo 'Hello World from Task!'
    deps:
      - run-first
  hello-works:
    cmds: [echo 'Hello World from Task!']
    deps:
      - run-first
  run-first: echo "This as a dependency should run first"
```
Gives the following output:
```sh
$ task hello-works -v
task: "hello-works" started
task: "run-first" started
task: [run-first] echo "This as a dependency should run first"
This as a dependency should run first
task: "run-first" finished
task: [hello-works] echo 'Hello World from Task!'
Hello World from Task!
task: "hello-works" finished
```

We would expect the `hello-broken` to either fail with a message that `cmd:` is an unsupported key or to behave similarly to the above output.
Actual output:
```sh
$ task hello-broken -v
task: "hello-broken" started
task: [hello-broken] echo 'Hello World from Task!'
Hello World from Task!
task: "hello-broken" finished
```

- Task version: 3.17.0
- Operating System: Ubuntu

**Comment by tennox** [comment]
I was also confused by this. Thought there is a nice short syntax for tasks with a single cmd (which would be my favorite), but then they just didn't do anything :sweat:

**Comment by pd93** [comment]
Hi @tennox and @Rots. Looks like this was fixed in #1131 and we forgot to close this issue. This change was released in v3.26.0, so updating to any version newer than this should fix this issue.
